plugins/drm/lib/api/KalturaDrmEntryContextPluginData.php

```diff
@@ -0,0 +1,13 @@
+<?php
+/**
+ * @package api
+ * @subpackage objects
+ */
+class KalturaDrmEntryContextPluginData extends KalturaPluginData{
+
+    /**
+     * @var string
+     */
+    public $flavorData;
```

Comment:
Since the attribute name is the most generic name in the world and explains nothing, please add a detailed description that explains it.